docker-compose.yml

```diff
@@ -29,14 +30,20 @@ services:
     depends_on:
       - base
       - postgres
+      - rabbit
     build:
       context: .
       dockerfile: docker/celery-beat/Dockerfile
     entrypoint: ["/app/docker/celery-beat/entrypoint.sh"]
+    environment:
+        CELERY_BROKER_URL: 'amqp://guest:guest@rabbit:5672'
     command: ["python3", '-m', "celery", "-A", "mysite", "beat", "-l", "info", "--scheduler", "django_celery_beat.schedulers:DatabaseScheduler"]
     tty: true
     volumes:
       - './django_celery_beat/:/app/django_celery_beat/'
 
+  rabbit:
+      image: rabbitmq:3.8.0
```

Comment:
is 3.8.1 available?

Reply:
Yes it is available, I just took this version from celery project. Do we need to restrict to some version? can we put latest tag ? so that always the latest stable version will be available in dev environment.

Reply:
ye can put latest tag

Reply:
I have added with latest tag.
@auvipy test case are failing, how to merge the code to master? I see that even it is failing for master branch or other PR as well